### DISCUSSION TITLE
Replace text in PDF

### DISCUSSION THREAD

**Discussion by Shakey11** [QuestPDF/QuestPDF#521] [General]
Hello, I need to replace a text in existing PDF (The text is text not a image). Is it possible to find and replce text with QuestPDF? Thank you.

**Comment by luis-fss** [comment]
This is not possible with QuestPDF, but you can use another library more oriented to pdf editing.

**Reply by niteshkejriwal** [reply]
Just checking, if it is possible now?

**Reply by luis-fss** [reply]
Nope.

**Reply by greident** [reply]
Just check, it is possible now ?

**Reply by luis-fss** [reply]
Not yet and (forgive the boldness) I don't think it will ever be.

QuestPDF is described by the author as "an open-source .NET library for PDF generation", it never offered anything related to PDF editing.

See the [About](https://github.com/QuestPDF/QuestPDF) section on the repository home page or, in more detail, in the [documentation](https://www.questpdf.com/).

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. As @luis-fss correctly explained, QuestPDF is a PDF generation library and does not support editing or modifying text in existing PDF files. For PDF editing needs, a dedicated library such as iText or PdfSharp would be more appropriate.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
